metadata/MachXO2/LCMXO2-1200HC/globals.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please keep these files in sync with prjtrellis-db, so that prjtrellis-db could theoretically be re-created from the prjtrellis repo.

Reply:
Actually, as you have a Python script to generated globals.json I think these could just be removed altogether